contracts/deposit-contract/depositContract.v.py

```diff
@@ -71,15 +88,12 @@ def deposit(deposit_input: bytes[512]):
     self.branch[i] = value
 
     self.deposit_count += 1
-
-    new_deposit_root:bytes32 = self.get_deposit_root()
-    log.Deposit(new_deposit_root, deposit_data, self.to_bytes8(index), self.branch)
+    new_deposit_root: bytes32 = self.get_deposit_root()
+    log.Deposit(new_deposit_root, deposit_data, self.to_little_endian_64(index), self.branch)
 
     if deposit_amount == self.MAX_DEPOSIT_AMOUNT:
         self.full_deposit_count += 1
         if self.full_deposit_count == self.CHAIN_START_FULL_DEPOSIT_THRESHOLD:
-            # Temporarily commenting out the day boundary. 
-            #timestamp_day_boundary: uint256 = deposit_timestamp - deposit_timestamp % SECONDS_PER_DAY + SECONDS_PER_DAY
-            #log.ChainStart(self.get_deposit_root(), self.to_bytes(timestamp_day_boundary))
-            log.ChainStart(self.get_deposit_root(), self.to_bytes8(deposit_timestamp))
-            self.chainStarted = True
+            timestamp_day_boundary: uint256 = as_unitless_number(block.timestamp) - as_unitless_number(block.timestamp) % SECONDS_PER_DAY + SECONDS_PER_DAY
```

Comment:
Can you comment out the day boundry as before and use the `deposit_timestamp` instead. It was commented out before because it broke our tests and validator flow.